.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 3.0.1
```

Comment:
Needless, you already uses .ruby-version

Reply:
Removed